src/lib/utils/toc.ts

```diff
@@ -29,18 +30,17 @@ export const parseHeadingId = (heading: string): string => {
 }
 
 /**
- * Parse out the title to be displayed in the Table of Contents by removing any custom
- * ID and Twemoji components, as well as any backslashes used to escape Markdown characters
+ * Parse out the title to be displayed in the Table of Contents
  * @param title Heading string without leading #s that may contain Emoji's or a {#custom-id}
  * @returns Title string with custom ID and Emoji's removed
  */
 export const parseToCTitle = (title: string): string => {
   const match = customIdRegEx.exec(title)
   const trimmedTitle = match ? title.replace(match[1], "").trim() : title
-  const sanitizedTitle = trimmedTitle
-    .replaceAll(emojiRegEx, "")
-    .replaceAll(emojiUnicodeRegEx, "")
-    .replaceAll(escapeSlashRegEx, "")
+  const sanitizedTitle = trimmedTitle.replaceAll(unicodeRegEx, (match) => {
+    const codePoint = match.replace(nonHexRegEx, "")
+    return String.fromCodePoint(parseInt(codePoint, 16))
```

Comment:
Looks for matches of `unicodeRegEx` (javascript unicode character syntax), then trims any non-hexadecimal characters and parsed the result as an int radix 16, and converts to character. This should work for any unicode characters, including emojis and non-standard character sets.

Reply:
Nevermind, this approach didn't actually work... Emoji's are parsed a little different so was unable to combine.